packages/web/src/components/nav/desktop/WalletsNestedContent.tsx

```diff
@@ -56,8 +56,8 @@ export const WalletsNestedContent = () => {
   }[tier]
 
   const audioBalanceFormatted = audioBalance
-    ? formatCount(audioBalance.div(WEI_DIVISOR).toNumber())
-    : '0'
+    ? AUDIO(audioBalance).toLocaleString()
```

Comment:
does this mess with the web BalancePill layout at all? i see that component doesn't specifically try to handle null 
https://github.com/AudiusProject/audius-protocol/blob/26bf15df43ece2bd283772d5c54815c7319e470b/packages/harmony/src/components/balance-pill/BalancePill.tsx#L32

should that be a loading state?